cardano-api/src/Cardano/Client/API.hs

```diff
@@ -0,0 +1,318 @@
+--                           , SendMsgRequestNextPipelined
+--                           , CollectResponse)
+    , ClientStNext (..)
+--    , chainSyncClientPeerPipelined -- cardano-explorer
+--    , ClientPipelinedStIntersect (..)
+-- Ouroboros.Network.Protocol.ChainSync.PipelineDecision
+    , pipelineDecisionLowHighMark
+    , PipelineDecision (..)
+    , runPipelineDecision
+    , MkPipelineDecision
+-- Ouroboros.Network.Protocol.ChainSync.Codec
+    , codecChainSync
+-- Ouroboros.Network.Protocol.ChainSync.Type
+    , ChainSync
+-- Ouroboros.Network.Protocol.LocalTxSubmission.Client
+    , LocalTxSubmissionClient (..)
+    , LocalTxClientStIdle (SendMsgSubmitTx)
+    , localTxSubmissionClientPeer
+-- Ouroboros.Network.Protocol.LocalTxSubmission.Codec
+    , codecLocalTxSubmission
+-- Ouroboros.Network.Protocol.LocalTxSubmission.Type
+    , LocalTxSubmission
+-- Network.TypedProtocol.Codec
+    , Codec
+-- Network.TypedProtocol.Codec.Cbor
+--    , DeserialiseFailure
+-- Network.TypedProtocol.Driver
+    , runPeer
+    , runPipelinedPeer
+-- Network.TypedProtocol.Pipelined
+    , Nat(Zero, Succ)
+-- Ouroboros.Consensus.Ledger.Byron
+    , ByronBlock (ByronBlock, byronBlockRaw, byronBlockSlotNo, byronBlockHash)
+    , ByronHash (..)
+    , GenTx
+-- Ouroboros.Consensus.Ledger.Abstract
+    , BlockProtocol
+-- Ouroboros.Consensus.Node.ProtocolInfo
+    , pInfoConfig
+    , protocolInfo
+-- Ouroboros.Consensus.Node.Run.Abstract
+    , RunNode
+    , nodeDecodeBlock
+    , nodeDecodeGenTx
+    , nodeDecodeHeaderHash
+    , nodeEncodeBlock
+    , nodeEncodeGenTx
+    , nodeEncodeHeaderHash
+    , nodeNetworkMagic
+-- Ouroboros.Consensus.Node.ErrorPolicy
+    , consensusErrorPolicy
+-- Ouroboros.Consensus.Protocol
+    , NodeConfig
+    , Protocol (..)
+
+-- wallet extra imports
+-- Codec.SerialiseTerm
+    , CodecCBORTerm
+-- Ouroboros.Network.Magic
+    , NetworkMagic (..)
+-- import qualified Ouroboros.Network.Block as O
+-- qualified Ouroboros.Network.Point as Point
+-- Ouroboros.Network.ChainFragment
+    , HasHeader (..) -- blockNo
+
+-- import Network.TypedProtocol.Channel
+    , Channel
+-- import Network.TypedProtocol.Driver
+    , TraceSendRecv
+-- import Ouroboros.Consensus.Ledger.Byron
+--    , ByronBlock (..)
+    , decodeByronBlock
+    , decodeByronGenTx
+    , decodeByronHeaderHash
+    , encodeByronBlock
+    , encodeByronGenTx
+    , encodeByronHeaderHash
+-- import Ouroboros.Network.NodeToClient
+    , ConnectionId (..)
+    , NetworkConnectTracers (..)
+    , NodeToClientVersion (..)
+    , connectTo
+    , localTxSubmissionClientNull
+-- import Ouroboros.Network.Protocol.ChainSync.Client
+    , ChainSyncClient (..)
+    , ClientStIdle (..)
+    , ClientStIntersect (..)
+    , chainSyncClientPeer
+-- import Ouroboros.Network.Protocol.Handshake.Version
+    , DictVersion (..)
+    , simpleSingletonVersions
+-- import Network.Mux.Interface
+-- import Network.Mux.Types
+--    , MuxError
+    )
+
+where
```

Comment:
I think it's just nicer to put the `where` just after the closing bracket.  That what we've been using in `ouroboros-network`.